tests/perf/test_accuracy.py

```diff
@@ -2,142 +2,255 @@
 # SPDX-License-Identifier: Apache-2.0
 
 import os
-from enum import Enum
+import random
+from pathlib import Path
+from time import time
 from typing import Any, Dict, List, Tuple
 
 import numpy as np
 import openvino as ov
+import pandas as pd
 import pytest
 
 from openvino_xai import Task
+from openvino_xai.common.parameters import (
+    BlackBoxXAIMethods,
+    Method,
+    Task,
+    WhiteBoxXAIMethods,
+)
 from openvino_xai.common.utils import retrieve_otx_model
 from openvino_xai.explainer.explainer import Explainer, ExplainMode
+from openvino_xai.explainer.explanation import Explanation
 from openvino_xai.explainer.utils import (
     ActivationType,
     get_postprocess_fn,
     get_preprocess_fn,
 )
+from openvino_xai.methods.black_box.base import Preset
 from openvino_xai.metrics import ADCC, InsertionDeletionAUC, PointingGame
-from tests.unit.explanation.test_explanation_utils import VOC_NAMES
+from tests.perf.perf_tests_utils import convert_timm_to_ir
+from tests.test_suite.custom_dataset import CustomVOCDetection
+from tests.test_suite.dataset_utils import (
+    DatasetType,
+    coco_anns_to_gt_bboxes,
+    define_dataset_type,
+    voc_anns_to_gt_bboxes,
+)
+from tests.unit.explainer.test_explanation_utils import VOC_NAMES, get_imagenet_labels
 
 datasets = pytest.importorskip("torchvision.datasets")
+timm = pytest.importorskip("timm")
+torch = pytest.importorskip("torch")
+
+
+IMAGENET_MODELS = [
+    "resnet18.a1_in1k",
+    # "resnet50.a1_in1k",
+    # "resnext50_32x4d.a1h_in1k",
+    # "vgg16.tv_in1k"
+]
+VOC_MODELS = [
+    # "mlc_mobilenetv3_large_voc"
+]
+TRANSFORMER_MODELS = [
+    "deit_tiny_patch16_224.fb_in1k",  # Downloads last month 8,377
+    # "deit_base_patch16_224.fb_in1k", # Downloads last month 6,323
+    # "vit_tiny_patch16_224.augreg_in21k",  # Downloads last month 3,671 - trained on ImageNet-21k
+    # "vit_base_patch16_224.augreg2_in21k_ft_in1k",  # Downloads last month 207,590 - trained on ImageNet-21k
```

Comment:
`vit_base_patch16_224.augreg2_in21k_ft_in1k` seems to be trained on 21k, but finetuned on 1k, right?

Reply:
I didn't check it, but I know that it can predict 21K classes, including 1K of ImageNet. We can use these models, but I have to support ImageNet21K label names (that is not a big deal actually)